meshnets/modules/processor.py

```diff
@@ -1,10 +1,40 @@
-""""Define the GraphProcessor class."""
+""""Define the Processor and ProcesserLayer classes."""
 
+import torch
+from torch.nn import Identity
+from torch_geometric.data import Batch
+from torch_geometric.nn.conv import MessagePassing
 
-class GraphProcessor:
 
-    def __init__(self, *args):
-        pass
+class MGNProcessorLayer(MessagePassing):
```

Comment:
Any reason for here the MessagePassing to be of convolutional type? What is the difference from `from torch_geometric.nn import MessagePassing`?

Reply:
Both imports refer to the same class. The implement I found used `torch_geometric.nn.conv` which is a bit more verbose, but the .conv is optional.